src/vm/exceptionhandling.cpp

```diff
@@ -362,14 +362,26 @@ void ExceptionTracker::UpdateNonvolatileRegisters(CONTEXT *pContextRecord, REGDI
         pAbortContext = GetThread()->GetAbortContext();
     }
 
+#ifndef FEATURE_PAL
```

Comment:
The comment above this method talks about FaultingExceptionFrame only. It looks like we can get here with HelperMethodFrame too ... could you please check the comment and update it as necessary?